src/univalue/include/univalue.h

```diff
@@ -180,7 +180,7 @@ class UniValue {
     bool push_back(std::pair<std::string,UniValue> pear) {
         return pushKV(pear.first, pear.second);
     }
-    friend const UniValue& find_value( const UniValue& obj, const std::string& name);
```

Comment:
univalue changes need to be submitted upstream, not here :)

Reply:
He did that (see OP).